dietpi/dietpi-update

```diff
@@ -238,11 +241,11 @@ Please select 'Update' option to apply the update."
 
 					G_WHIP_YESNO "
 ------------------------Notice------------------------
-The benefit of DietPi is we use standard linux configurations and commands. The downside is we can't possibly accommodate or predict, every modifcation to Linux configurations files by the end user, during the update.
-Although we test the updates thoroughly, if you have made custom changes to Linux configuration files, outside of the DietPi programs, an update may trigger a potential issue.
+A benefit of DietPi is, that we use standard Linux (Debian) configurations and commands. The downside is, that we can't possibly accommodate or predict every modifcation to Linux configurations files by the end user, during the update.
```

Comment:
Doesn't work for me 😄 

You could do
```
- A benefit of DietPi is: We use standard Linux (Debian) configurations and commands.
- A potential downside is: We can't possibly accommodate or predict all modification to Linux configurations files by the end user, outside of DietPi programs, during updates.
```

Reply:
I'll make the changes.

Reply:
@Fourdee 
Checking `deepl.com`, yes you are right, the additional `, that` made it worse 😅.
Actually I just found `The benefit` a bid too pronounced, since we have so much benefits.
Your solution looks more structured, nice as well 👍.